app/views/posts/show.html.erb

```diff
@@ -1 +1,32 @@
-<h2> Posts <%= params[:id] %>for given user </h2>
\ No newline at end of file
+<h2 class='heading'> Posts for user number <%= params[:id] %></h2>
+<div class='post-details-body'>
+    <div class='post-details-p-card'>
+        <div class='post-details-top'>
+            <h3>
+                <%= @post.title %> by <%= @post.user.name %>
+            </h3>
+
+            <div class='post-com-lik-counters'>
```

Comment:
- [ ] Kindly not `<%= @post.user.name %>` this relationship does not exist in your table, if you want to get the user name kindly refer to the `<%= @user.name %>` which is defined in the post#show controller. I am getting this error :point_down: 

![image](https://user-images.githubusercontent.com/65068771/156240033-d5af431a-9dac-4d15-aa85-009289dba199.png)